fedbiomed/common/models/_sklearn.py

```diff
@@ -399,6 +399,32 @@ def export(self, filename: str) -> None:
         with open(filename, "wb") as file:
             joblib.dump(self.model, file)
 
+    def reload(self, filename: str) -> None:
+        """Import and replace the wrapped model from a dump file.
+
+        Args:
+            filename: path to the file where the model has been exported.
+
+        !!! info "Notes":
+            This method is designed to load the model from a local dump
+            file, that might not be in a trustworthy format. It should
+            therefore only be used to re-load data exported locally and
+            not received from someone else, including other FL peers.
+
+        Raises:
+            FedbiomedModelError: if the reloaded instance is of unproper type.
+        """
+        model = self._reload(filename)
```

Comment:
should we catch here os errors ? and convert them into fedbiomed error (such as FileNotFoundError, MemoryError, PermissionError)

Reply:
also should we have an error if the file already exists?